### PR TITLE
Smart broadcast_to matcher

### PR DESCRIPTION
**Pull request by TheSmartnik**
Hi,

PR regarding https://github.com/palkan/action-cable-testing/issues/9.
What do you think?

### CODE REVIEW

**Comment by TheSmartnik** [comment]
Oh wait, I thought of a better way to do it. Gonna push soon

**Comment by palkan** [comment]
@TheSmartnik LGTM 👍 

Let's add a scenario to `features/matchers/have_broadcasted_matcher.feature` and a change log entry.

**Comment by TheSmartnik** [comment]
@palkan Done

---
I don't have much experience with cucumber though and  was interested why are you using cucumber here? What advantages over regular rspec/minitest testing does it give here?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Change log
 
+## 0.1.3 (2017-11-27) ([@TheSmartnik][])
```
Comment (palkan) [inline CHANGELOG.md]: You have to add a link to the end of the file in order it to be resolvable

**Comment by palkan** [comment]
> I don't have much experience with cucumber though and was interested why are you using cucumber here? What advantages over regular rspec/minitest testing does it give here?

Good question) The only reason I'm using cucumber here is to be compatible with `rspec-*` development style (in order to make it easier to merge this into `rspec-rails` later). And they use it to build [relish docs](https://relishapp.com/palkan/action-cable-testing/docs).

File: CHANGELOG.md (hunk 1 above)
Comment (palkan) [reply CHANGELOG.md]: And let's use `master` version and do not update the version itself.

Comment (TheSmartnik) [reply CHANGELOG.md]: > You have to add a link to the end of the file in order it to be resolvable

Thanks 👍  Didn't know that

**Comment by TheSmartnik** [comment]
> Good question) The only reason I'm using cucumber here is to be compatible with rspec-* development style (in order to make it easier to merge this into rspec-rails later). And they use it to build relish docs.

Oh, cool. Thanks for explaining 😄

**Comment by TheSmartnik** [comment]
@palkan All good now

**Comment by palkan** [comment]
Thanks, @TheSmartnik!

I gonna release as soon as I implement the same functionality for Minitest assertions.

**Pull request merged**